drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -283,6 +283,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     if (!window.gettext) { window.gettext = function gettext_stub(string) { return string; }; }
 
     var $element = $(element);
+    element = $element[0]; // <- workaround Studio bug until https://github.com/edx/edx-platform/pull/11433 is merged
```

Comment:
It looks like the edx-platform PR merged, so I don't think you need this workaround anymore.